net-libs/libomemo-c/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
This is forbidden,
https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602

Would you like to maintain this library and become a co-/main maintainer with qxmpp? Otherwise we'd need @0xd34df00d to ack this PR and become a maintainer for this dep. To me it looks like these packages may need to be dropped to maintainer-needed if no one is maintaining them. Judging from the jump in versions with qxmpp, it looks rather neglected.

Reply:
Yes, I can maintain these packages

Reply:
Added to metadata